pkg/machines/manifest.json.in

```diff
@@ -9,5 +9,6 @@
         "path": "index.html",
         "order": 60
      }
-  }
+  },
+  "content-security-policy": "default-src 'self' 'unsafe-inline' data:"
```

Comment:
Please suggest what's the best place to document this.
So far I put following to the PR description (see above):

    The cockpit-machines newly sets
    "content-security-policy": "default-src 'self' 'unsafe-inline' data:"
    to make the no-vnc and download of dynamically generated files (the .vv) working.